source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -196,6 +196,12 @@ namespace
         printResults({ "  Suggest settings time:" }, durations);
     }
 
+    void printcopyDataResults(const std::vector<Duration> &durations)
+    {
+        printSecondarySeparationLine();
+        std::cout << "  copyData time : \t " << formatDuration(durations.at(0)) << std::endl << std::endl;
```

Comment:
`Copy data` instead of `copyData `

Reply:
I actually meant this:
`std::cout << "  copyData time :` to `std::cout << "  Copy data time :`
But what you changes is also good.